Services/Seed.cs

```diff
@@ -147,51 +147,30 @@ public class Seed : ISeed
 
```

Comment:
Please add a log statement here for "Creating Seed Rules."

Reply:
Fixed. The log message "Create Seed Device group" added.

For the log message about the whole process start, it was already there in Seed.TrySeedAsync, line 71